src/behaviours/store/connect.js

```diff
@@ -69,9 +69,20 @@ export default function connectToStores(storesConfiguration, getState) {
                 });
             }
 
+			componentDidMount(){
+				this._isMounted = true;
```

Comment:
Nice to respect the good react pattern https://facebook.github.io/react/blog/2015/12/16/ismounted-antipattern.html
Thanks!